bin/branding.list

```diff
@@ -10,6 +10,7 @@ AUTHOR
 LICENSE
 HOMEPAGE
 CHANGELOG_PAGE
+ORGANIZATION
```

Comment:
@jtnord  Never mind: I forgot the test targets are totally isolated from the others, so we won't hit failures in branding if the variable is not set and not used.   Publish will run until hitting an error where it's used as a normal variable... rather than either failing in branding or earlier. :8ball: 

Right now in my ideal fantasyland, each of our separate build paths could run as independently as possible with minimal variables required, and fail up front if any are undefined.  But that's in the land of rainbows and unicorn farts.